### ISSUE TITLE
Wallet.fromSeed not working

### ISSUE THREAD

**Issue by gutenye** [input-output-hk/js-cardano-wasm#39]
Given

```js
const seed = '0'.repeat(32).split('')
CardanoCrypto.Wallet.fromSeed(seed)
```

it returns `false`

**Comment by NicolasDP** [comment]
It seems to me that the returned value of this call is:

```javascript
> '0'.repeat(32).split('')
(32) ["0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0", "0"]
```

However we are expecting an array of _bytes_, more like this:

```javascript
> Array(32).fill(0)
(32) [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
```

**Comment by gutenye** [comment]
@nicarq 
Thanks for quick reply. Tested with `Array(32).fill(0)`, still returns `false`.
My version is built from the master branch.

**Comment by nicarq** [comment]
@gutenye wrong Nicolas :)

**Comment by NicolasDP** [comment]
I'll put that up on my task list

**Comment by NicolasDP** [comment]
thanks for reporting the issue, it was indeed a bug. The SEED_SIZE was not exported and was left `undefined`. We then always had an invalid Wallet's SEED. This is fixed in #40 . I'll merge the PR which will close this issue. However if you find the issue is not solved to satisfaction, we can always reopen.
